### DISCUSSION TITLE
Not working in x86

### DISCUSSION THREAD

**Discussion by Raykler** [vvvv/SVG#858] [Q&A]
I created a dll to manage SVG files and retrieve Image or Icon from them.
It works properly, but only in x64, when I try to run an x86 app, the app doesn't run.

When debugging, if I swap between AnyCPU and x86, the icons dissapear.
![AnyCPU](https://user-images.githubusercontent.com/8381731/105868056-0a0ab700-5ff6-11eb-8193-c7cd9cf81b0f.png) ![x86](https://user-images.githubusercontent.com/8381731/105868057-0aa34d80-5ff6-11eb-94fc-3046bd76296a.png)

This is one of the methods that i use:
```
public static Image LoadImage(string file, int width, int height)
{
    if (!File.Exists(file) || height < 1 || width < 1)
        return new Bitmap(width, height);

    FileInfo FInfo = new(file);
    if (FInfo.Extension.ToUpper() != ".SVG")
        return new Bitmap(width, height);

    XmlDocument svg = new();
    svg.Load(file);

    return LoadImage(svg, width, height);
}
```

```
public static Image LoadImage(XmlDocument svg, int width, int height)
{
    if (height < 1 || width < 1)
        return new Bitmap(width, height);

    SvgDocument SVGFile = SvgDocument.Open(svg);
    SVGFile.Height = height;
    SVGFile.Width = width;

    return SVGFile.Draw();
}
```

I'm using .Net 5, and I've tested it with NuGet versions 3.1.1 and 3.2.3.

Is it any way to fix it?

**Comment by H1Gdev** [comment]
@Raykler 

I created and checked a minimal app, but it displayed correctly on `Any CPU`, `x64`, and `x86`.

- TargetFramework: net5-windows
- Svg: 3.2.3

[WindowsFormsApp1.zip](https://github.com/svg-net/SVG/files/5876908/WindowsFormsApp1.zip)

**Comment by Raykler** [comment]
I've finally found the problem.

I've tested it with an empty project (also Console project with both _Net 5_ and _Net Core 3.1_), and it doesn't work in x86 if you compile for AnyCPU. It seems that Net Core doesn't support the AnyCPU deployment. 

![error](https://user-images.githubusercontent.com/8381731/105970521-c06bac00-6089-11eb-8257-80111fa05f8f.png)

**Comment by H1Gdev** [comment]
@Raykler 

This seems to be helpful to you.
https://stackoverflow.com/questions/62773887/net-core-3-1-console-app-will-not-run-on-windows-7
